### DISCUSSION TITLE
Sliderends (Small Ticks) are missing visual feedback

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#18907] [Q&A] [answered]
While reading through the issues recently created by peppy (https://github.com/ppy/osu/issues/18893 in particular) I remembered that Sliderends are still missing feedback. I'm not sure if it fits the issues description so I decided to open a discussion instead. I could have sworn that there was already an issue tracking this, but I couldn't find anything apart from https://github.com/ppy/osu/discussions/15977 maybe, but that was more talking about 100s/50s so not sure.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/8700

**Reply by respektive** [reply]
right i didn't search for tails, just ends... this can probably be marked as answered then.
